src/proto/protocached.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package protocached;
+
+// Result for rpc response
+message ResponseResult {
+  enum ResultType {
+    SUCCESS = 0;
+    ERROR = 1;
+  }
+
+  // specify the response result
+  ResultType result = 1;
```

Comment:
Maybe change the name of the field to result_type or result_status  if you decide to change the message type name to ResultStatus?